### ISSUE TITLE
Sprite strange behaviour when overlapping walls

### ISSUE THREAD

**Issue by Aranclanos** [unitystation/unitystation#1779]
# Description
![image](https://user-images.githubusercontent.com/701959/56595024-41983400-65c4-11e9-910c-cf807630ffb4.png)
If an object is on top of a out of vision tile, but its sprite overalps a wall, the part of the sprite that is on top of the wall will be visible
Notice how the beef jerky on this image is half-visible where it shouldn't be visible at all

## Category
BUG

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1.Head to escape and use the vending machine or throw an object against a wall
2.Make sure the sprite of the item overlaps the wall at least slightly
3.Walk out of vision of the tile but not the wall
4.See half the object visible (the part of the sprite overlapping the wall)

**Comment by fomalsd** [comment]
yeah, might consider stopping items half tile earlier, at the moment the only solid part is item's center so its sprite can overlap walls
OR
modify items' layer order so that they are below walls
